packages/cli/jest.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   transform: {
-    '^.+\\.ts$': '@swc/jest',
+    '^.+\\.(m?j|t)s$': '@swc/jest',
   },
+  transformIgnorePatterns: ['@prisma'],
```

Comment:
This excludes re-transpiling our own files because we know they are already cjs. This statement also removes the implicit exclusion of `node_modules`, meaning that we transpile `node_modules` too. That allows us to upgrade to esm packages.